data/xml/schema.rnc

```diff
@@ -9,7 +9,8 @@ MarkupText = (text | b | i | url | fixed-case | tex-math )+
 
 first = element first { text }
 last = element last { text }
-Person = attribute id { xsd:NCName }?, (first? & last)
+Variant = element variant { (first? & last) }
```

Comment:
Should `<variant>` be allowed to have a `lang` or `script` attribute?

Reply:
Yes, good idea. I can see arguments for both:

- `lang` provides more information and seems more direct, and languages typically have a canonical script (I think?)
- `script` seems appropriate, since the most likely use case is to write a name with a non-Latin script, and also there is no necessary link between a script and a language

I think I'm inclined towards `lang` but am ambivalent.

Reply:
I'm inclined towards script :) I think the more common case is to preserve a name's appearance (rather than sound or meaning) across languages, e.g., a paper written Italian would still cite me as David Chiang, not Davide Chiang (right?) and so it's not clear in what sense my name is "English."

Reply:
That's the heart of it, you've sold me. What about the name decomp issue I mentioned below?